mev_inspect/punks.py

```diff
@@ -0,0 +1,104 @@
+from typing import List
+from mev_inspect.schemas.traces import (
+    ClassifiedTrace,
+    Classification,
+    DecodedCallTrace,
+)
+from mev_inspect.schemas.punk_bid import PunkBid
+from mev_inspect.schemas.punk_accept_bid import PunkBidAcceptance
+from mev_inspect.schemas.punk_snipe import PunkSnipe
+from mev_inspect.traces import get_traces_by_transaction_hash
+
+
+def get_punk_snipes(
+    punk_bids: List[PunkBid], punk_bid_acceptances: List[PunkBidAcceptance]
+) -> List[PunkSnipe]:
+    punk_snipe_list = []
+
+    for punk_bid in punk_bids:
+        for punk_bid_acceptance in punk_bid_acceptances:
+            if punk_bid.punk_index == punk_bid_acceptance.punk_index:
+                if punk_bid.amount > punk_bid_acceptance.min_price:
+                    punk_snipe = PunkSnipe(
+                        block_number=punk_bid.block_number,
+                        transaction_hash=punk_bid.transaction_hash,
+                        trace_address=punk_bid.trace_address,
+                        from_address=punk_bid.from_address,
+                        punk_index=punk_bid.punk_index,
+                        min_acceptance_price=punk_bid_acceptance.min_price,
+                        acceptance_price=punk_bid.amount,
```

Comment:
Using price and amount interchangeably introduces unnecessary fuzz. Would be clearer if either only price or only amount were used consistently

Reply:
Good call out